geopy/geocoders/what3words.py

```diff
@@ -53,7 +52,9 @@ def __init__(
 
         :param string scheme: Use 'https' or 'http' as the API URL's scheme.
             Default is https. Note that SSL connections' certificates are not
-            verified.
+            verified.  As of API v2 only 'https' is supported - this parameter
+            remains for backwards compatibility (setting to 'http' will stop
+            this geocoder working).
```

Comment:
I suggest to explicitly raise an exception in case `http` is passed.

```
if scheme == "http":
    raise ConfigurationError("What3Words doesn't support `http` scheme anymore. Please use `https` instead.")
```